src/test/java/com/uber/cadence/workflow/WorkflowTest.java

```diff
@@ -6251,4 +6260,43 @@ public void testEnqueueWorkflow() throws Exception {
 
     assertEquals("value", stub.getResult(String.class));
   }
+
+  public interface ActivityThrowsErrorWorkflow {
+
+    @WorkflowMethod
+    void execute();
+  }
+
+  public static class ActivityThrowsErrorWorkflowImpl implements ActivityThrowsErrorWorkflow {
+
+    public ActivityThrowsErrorWorkflowImpl() {}
+
+    @Override
+    public void execute() {
+      TestActivities activity =
+          Workflow.newActivityStub(
+              TestActivities.class,
+              new ActivityOptions.Builder()
+                  .setTaskList(Workflow.getWorkflowInfo().getTaskList())
+                  .build());
+      activity.throwIOError();
+    }
+  }
+
+  @Test
+  public void testActivityThrowsError() throws Exception {
```

Comment:
This was mostly for my own curiosity since we have special handling for this case and currently no test coverage. We intentionally don't report Errors, which I'm not convinced is a good or useful thing.

Reply:
given the severity of what `Error` generally represents, tbh I think "have no special handling, let it tear things apart as it goes" is likely best.